vm/systemSmartContracts/governance.go

```diff
@@ -0,0 +1,872 @@
+//go:generate protoc -I=proto -I=$GOPATH/src -I=$GOPATH/src/github.com/gogo/protobuf/protobuf  --gogoslick_out=. governance.proto
+package systemSmartContracts
```

Comment:
missing unit tests. File very large. Parts of it (especially checks of arguments) contains duplicate code with other system contracts

Reply:
unit tests will come in next PR - checks are a must and every function has it - but each of them is specific.